docs/installation.rst

```diff
@@ -125,6 +117,13 @@ if you run ``pixi update`` or change the ``pixi.toml`` file. Both the
 ``pixi.toml`` as well as the ``pixi.lock`` are required to fully reproduce an
 environment. 
 
+You can activate your environment using::
+
+  pixi shell
```

Comment:
you can also directly start it by using "pixi run code"

If you want to start vscode in the current directory that would be "pixi run code ."